builder/virtualbox/common/step_port_forwarding.go

```diff
@@ -34,6 +36,58 @@ type StepPortForwarding struct {
 	l *net.Listener
 }
 
+var vboxVerMinNeedLHReachable string
+
+func init() {
+	const vboxVerMinNeedLHReachableNonCanon = "v7.0"
+	var err error
+	vboxVerMinNeedLHReachable, err = getCanonicalSemver(vboxVerMinNeedLHReachableNonCanon)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Plugin failure: %v\nPlease open a Github issue for this plugin.\n", err)
+		os.Exit(1)
+	}
+}
+
+func getCanonicalSemver(version_string string) (string, error) {
+	canonical_semver := semver.Canonical(version_string)
+	if canonical_semver == "" {
+		return "", fmt.Errorf("String '%s' is not a valid semver", version_string)
```

Comment:
Instead of `'%s'` for quoted strings, you can use `%q` for this purpose

```suggestion
		return "", fmt.Errorf("String %q is not a valid semver", version_string)
```